data/data/bootstrap/bootstrap-in-place/files/usr/local/bin/bootstrap-in-place.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -euoE pipefail ## -E option will cause functions to inherit trap
+
+# This script is executed by bootkube.sh when installing single node with bootstrap in place
+CLUSTER_BOOTSTRAP_IMAGE=$1
+
+
+bootkube_podman_run() {
+  # we run all commands in the host-network to prevent IP conflicts with
+  # end-user infrastructure.
+  podman run --quiet --net=host "${@}"
+}
+
+if [ ! -f stop-etcd.done ]; then
+  echo "Stop etcd static pod by moving the manifest"
+  mv /etc/kubernetes/manifests/etcd-member-pod.yaml /etc/kubernetes
```

Comment:
Seems like there is a perma-fail possibility here where we successfully `mv` here but time out or otherwise hiccup while waiting for etcd to go down.  Then, because we died before the `stop-etcd.done` touch, we come into this block again, try to `mv` again, and fail, and continue to fail on the `mv` in all future unit/script invocations.  Can we:

```sh
test -f /etc/kubernetes/manifests/etcd-member-pod.yaml && mv /etc/kubernetes/manifests/etcd-member-pod.yaml /etc/kubernetes
```

or:

```sh
mv /etc/kubernetes/manifests/etcd-member-pod.yaml /etc/kubernetes || echo "already moved etcd-member-pod.yaml
```

or something?